p5/requirements.txt

```diff
@@ -23,6 +23,7 @@ pydocstyle==6.0.0
 pylint==2.7.4
 pyparsing==2.4.7
 pytest==6.2.2
+pytest-mock==3.6.1
```

Comment:
I was expecting to see more lib upgrades.  For example, in a test virtual env, I'm getting `pylint 2.11.1`.